src/test/java/com/wikia/webdriver/testcases/desktop/createawikitests/CreateWikiTestsLoggedInUser.java

```diff
@@ -1,123 +1,127 @@
 package com.wikia.webdriver.testcases.desktop.createawikitests;
 
+import static com.wikia.webdriver.common.core.Assertion.assertStringContains;
+
 import com.wikia.webdriver.common.contentpatterns.CreateWikiMessages;
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.api.WikiFactory;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.core.helpers.WikiaProperties;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.elements.communities.desktop.pages.CreateNewWikiPage;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep1;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep2;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.createnewwiki.CreateNewWikiPageObjectStep3;
+
 import org.testng.annotations.Test;
 
 @Test(groups = {"CNW_User"})
+@Execute(onWikia = "community")
 public class CreateWikiTestsLoggedInUser extends NewTestTemplate {
 
-  private String wikiDomain;
-
   @Test(groups = {"CNW", "CreateNewWikiLoggedIn_001"})
```

Comment:
Maybe we can resign from those `"CreateNewWikiLoggedIn_001"` and use the name of the test?